posts/MigueTripi/01.isContractIssue.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Using OpenZeppelin standard IsContract() could be unsafe'
+title: 'Why using OpenZeppelin's IsContract() may be unsafe'
```

Comment:
```suggestion
title: "Why using OpenZeppelin's IsContract() may be unsafe"
```

Reply:
You need to use double quotes in order to use a single quote inside it

Reply:
Done